doc/release/1.7.0-notes.rst

```diff
@@ -24,53 +24,290 @@ For running on PyPy, PyPy3 6.0+ is required.
 Highlights of this release
 --------------------------
 
--
+- A new submodule for quasi-Monte Carlo, `scipy.stats.qmc`, was added
+- The documentation design was updated to use the same PyData-Sphinx theme as
+  other NumFOCUS packages like NumPy.
+- We now vendor and leverage the Boost C++ library to enable numerous
+  improvements for long-standing weaknesses in `scipy.stats`
+- ``cdist`` and ``pdist`` distance calculations are faster for several metrics,
+  especially weighted cases, thanks to a rewrite to a new C++ backend framework
```

Comment:
I suggest adding `RBFInterpolator` as a highlight.

@treverhines would you mind suggesting some working, also in the `scipy.interpolate` improvements section further down?

Reply:
suggested highlight:
```
- A new class for radial basis function interpolation, `RBFInterpolator`, was added to address issues with the `Rbf` class.
```

Reply:
added the highlight, thank you